format/spec.md

```diff
@@ -513,7 +514,6 @@ Manifests for a snapshot are tracked by a manifest list.
 
 Valid snapshots are stored as a list in table metadata. For serialization, see Appendix C.
 
-
```

Comment:
Can you revert this unnecessary whitespace change?

Reply:
thanks for pointing this out, fixed